frontend/src/types/custom.d.ts

```diff
@@ -3,6 +3,8 @@
 // Allow svg imports in .tsx
 // https://stackoverflow.com/questions/44717164/unable-to-import-svg-files-in-typescript
 declare module '*.svg' {
+  import React = require('react');
+  export const ReactComponent: React.FC<React.SVGProps<SVGSVGElement>>;
```

Comment:
are these needed?

Reply:
Yep, to enable importing SVG element at compilation time :)